project/Dependencies.scala

```diff
@@ -137,8 +143,16 @@ object Dependencies {
     lazy val faDbSlick = "za.co.absa.fa-db" %% "slick" % Versions.fadb
     lazy val slickPg = "com.github.tminglei" %% "slick-pg" % Versions.slickpg
 
+    // awssdk library for providing the functionality of retrieving the secret key
+    lazy val awsSdk = "software.amazon.awssdk" % "secretsmanager" % Versions.awssdk
+    lazy val sso = "software.amazon.awssdk" % "sso" % Versions._sso
+
+    // For allowing array of strings to be parsed as json
+    lazy val spray = "io.spray" %% "spray-json" % Versions.spray
```

Comment:
What's the reason to introduce yet another json library into the project?

Reply:
That will be removed. It was before I realize that I didn't even need convert the response to json.